### PR TITLE
sahibinden gallery update

### PR DESCRIPTION
**Pull request by metehancelik**

### CODE REVIEW

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig [hunk 1]
```diff
@@ -11,34 +11,42 @@
...
         {% endfor %}
     </div>
 
 {% endblock %}
 
+{% block styles %}
+    <style>
+        {{ asset_inline('visiosoft.theme.base::css/theme.css') }}
+    </style>
+    {{ asset_style('visiosoft.module.advs::css/list.css') }}
```
Comment (fatihalp) [inline addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:46]: why these files? really needed?

**Comment by fatihalp** [comment]
which themes or devices tested?

sahibinden
gg
alibaba


web
tablet
android phone
ios phone

@metehancelik

**Comment by metehancelik** [comment]
for responsive, all devices are tested. for themes, only base theme is tested. gonna test other themes asap. @fatihalp

**Comment by metehancelik** [comment]
all themes are tested now, seems no problem.

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig (hunk 1 above)
Comment (metehancelik) [reply addons/default/visiosoft/advs-module/resources/views/list/gallery.twig]: changed to this:
{% block styles %}
    {{ asset_style('visiosoft.theme.base::css/gallery.css') }}
    {{ asset_style('visiosoft.module.advs::css/list-new.scss') }}
{% endblock %}

File: addons/default/visiosoft/base-theme/resources/css/gallery.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
+.product-hb {
+    font-family: -apple-system,
```
Comment (muammertop) [inline addons/default/visiosoft/base-theme/resources/css/gallery.css:2]: Do you need these fonts? are we available?

**Review by muammertop — CHANGES_REQUESTED** [review]

File: addons/default/visiosoft/base-theme/resources/css/gallery.css (hunk 2 above)
Comment (fatihalp) [reply addons/default/visiosoft/base-theme/resources/css/gallery.css:2]: these codes written by @YosefGul 

but these are not needed @metehancelik

Comment (metehancelik) [reply addons/default/visiosoft/base-theme/resources/css/gallery.css:2]: We already had those blocks in a different file which is in advs-module and I do not remember the name of it now. I just moved them in order to be more systematical.

Comment (metehancelik) [reply addons/default/visiosoft/base-theme/resources/css/gallery.css:2]: @muammertop @fatihalp I checked the 'font' lines, they are not needed. Will delete and commit again.

**Review by muammertop — APPROVED** [review]

**Review by spektra2147 — APPROVED** [review]

**Pull request merged**